UnityProject/Assets/Scripts/Objects/WallmountBehavior.cs

```diff
@@ -39,6 +40,14 @@ public bool IsFacingPosition(Vector3 worldPosition)
 		return difference >= 91 || difference <= -91;
 	}
 
+	//[DrawGizmo(GizmoType.Selected | GizmoType.NonSelected)]
+	void OnDrawGizmos ()
+	{
+		Gizmos.color = Color.green;
+        //Gizmos.DrawSphere(transform.position , 3);
+		Gizmos.DrawLine(transform.position,(transform.position-transform.up ));
```

Comment:
FYI, there's `GizmoUtils.DrawArrow` method that draws an actual arrow

Reply:
yeah, I was looking for that but everywhere said it wasn't inbuilt, I can add it now

Reply:
yep, check an example from PushPull:
```cs
GizmoUtils.DrawArrow( transform.position, PulledBy.transform.position - transform.position, 0.1f );
```
last parameter is arrowHeadLength, you can keep it at 0.1

Reply:
the Default seems to be good, 
Since that will be the exact tile that the game uses for calculations

Reply:
try applying this
```suggestion
	GizmoUtils.DrawArrow( transform.position, (transform.position-transform.up), 0.1f ); 
```